src/dateUtils.js

```diff
@@ -1,44 +1,51 @@
 const { DateTime, Duration } = require('luxon');
 
 const toDate = val => {
+  let t;
   if (DateTime.isDateTime(val)){
-    return val;
-  } else if (typeof val === 'number'){
-    return DateTime.fromMillis(val);
-  } else if (typeof val === 'string'){
-    if (val.match(/^[0-9]{4}-[0-9]{2}-[0-9]{2}$/)){
-      return DateTime.fromISO(val);
-    } else if (val.match(/^[0-9]{2}\/[0-9]{2}\/[0-9]{4}$/)){
-      return DateTime.fromFormat(val, 'dd/MM/yyyy');
-    }
-    return DateTime.fromRFC2822(val);
+    t = val;
   } else if (typeof val === 'object'){
```

Comment:
DateTime is still an object type

Reply:
i don't understand this condition...

i still feel your unit tests are lacking here. nice tests using `setNow` but I think you'd also benefit from just a unit test that throws a bunch of inputs at this function and validates the expected output.  the input has so much variability, there are a lot of scenarios

Reply:
Updated & more tests added